app/views/pages/_header.html.erb

```diff
@@ -0,0 +1,5 @@
+<nav class="navbar navbar-expand-lg bg-success border-bottom border-body" data-bs-theme="dark">
+  <div class="container-fluid">
+    <%= link_to "BlogApp", root_path, class:"navbar-brand"%>
+  </div>
+</nav>
```

Comment:
- [ ] [OPTIONAL] Good job adding the devise authentication in your blog app, everything about the Registration and the Login works properly. At the same time , I was able to detect an issue when I wanted to create a new post ,your bug is supposed to be working , 👍 Kindly fix that issue to allow users to create posts 😄 

![Capture1](https://github.com/YoutLone/Blog-App/assets/80458199/75031c5e-1ad8-4c09-9998-d8a08144d458)

![Capture](https://github.com/YoutLone/Blog-App/assets/80458199/e4a48e11-7b89-44d0-96a1-9c8191e7f626)

Reply:
Thank you @iambenkis for your review and I will fix it.